sql/src/test/java/io/crate/integrationtests/AnyIntegrationTest.java

```diff
@@ -31,13 +30,8 @@
 
 import static org.hamcrest.Matchers.is;
 
-@CrateIntegrationTest.ClusterScope(scope = CrateIntegrationTest.Scope.GLOBAL)
 public class AnyIntegrationTest extends SQLTransportIntegrationTest {
 
-    static {
-        ClassLoader.getSystemClassLoader().setDefaultAssertionStatus(true);
```

Comment:
imho assertions should be enabled in the junit configuration. I'll make another PR to update the idea configuration in the build.gradle so that that is done automatically on gradlew idea